Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidgetButtonMegaphoneIconView.swift

```diff
@@ -0,0 +1,174 @@
+import UIKit
+
+class SentryUserFeedbackWidgetButtonMegaphoneIconView: UIView {
+    init(config: SentryUserFeedbackConfiguration) {
+        super.init(frame: .zero)
+    
+        let svgLayer = CAShapeLayer()
+        svgLayer.path = megaphoneShape
+        
+        func configureLightTheme() {
+            svgLayer.fillColor = config.theme.foreground.cgColor
+        }
+        if #available(iOS 12.0, *) {
+            if UIScreen.main.traitCollection.userInterfaceStyle == .dark {
+                svgLayer.fillColor = config.darkTheme.foreground.cgColor
+            } else {
+                configureLightTheme()
+            }
+        } else {
+            configureLightTheme()
+        }
+        
+        svgLayer.fillRule = .evenOdd
+        
+        layer.addSublayer(svgLayer)
+        translatesAutoresizingMaskIntoConstraints = false
+        
+        if config.scaleFactor != 1 {
+            layer.transform = CATransform3DMakeScale(config.scaleFactor, config.scaleFactor, 0)
+        }
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    //swiftlint:disable function_body_length
+    lazy var megaphoneShape: CGPath = {
```

Comment:
l: I love the idea of drawing like this.

You can achieve a very similar result with the following code:

```Swift
lazy var megaphoneShape: CGPath = {
        let path = CGMutablePath()
        
        path.move(to: CGPoint(x: 1, y: 3))
        path.addLine(to: CGPoint(x: 7, y: 3))
        path.addLine(to: CGPoint(x: 10, y: 1))
        path.addLine(to: CGPoint(x: 12, y: 1))
        path.addLine(to: CGPoint(x: 12, y: 11))
        path.addLine(to: CGPoint(x: 10, y: 11))
        path.addLine(to: CGPoint(x: 7, y: 9))
        path.addLine(to: CGPoint(x: 1, y: 9))
        path.closeSubpath()
        
        path.addRect(CGRect(x: 2, y: 9, width: 3.5, height: 6))
        
        path.move(to: CGPoint(x: 12, y: 6))
        path.addRelativeArc(center: CGPoint(x: 12, y: 6), radius: 3, startAngle: -(.pi/2), delta: .pi)
        
        return path
    }
```

Reply:
I will let @Jesse-Box or @vuluongj20 comment on this.

Reply:
I think i'm just gonna go with the simpler version and if anyone doesn't like it, we can always revert. Thanks @brustolin